chalk-parse/src/ast.rs

```diff
@@ -68,6 +68,23 @@ pub enum Parameter {
     Lifetime(Lifetime),
 }
 
+pub struct TraitBound {
```

Comment:
Maybe this should not be called `TraitBound`? I would think something like `TypeParameterBound`